src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -0,0 +1,27 @@
+from argparse import ArgumentParser
+
+import pytest
+
+from diffpy.labpdfproc.tools import load_user_metadata
+
+params5 = [
+    ([[]], []),
+    ([["toast=for breakfast"]], [["toast", "for breakfast"]]),
+    ([["mylist=[1,2,3.0]"]], [["mylist", "[1,2,3.0]"]]),
+    ([["weather=rainy", "day=tuesday"]], [["weather", "rainy"], ["day", "tuesday"]]),
+]
+
+
+@pytest.mark.parametrize("inputs, expected", params5)
+def test_load_additional_info(inputs, expected):
+    actual_parser = ArgumentParser()
```

Comment:
Since this one is a bit more tricky, I htink it is better if we do actually test the parser in this case, so actually run the parser and don't just create a new namespace.

For example, i know htat your "toast", "for breakfast" passes the tests, but what do I actually type on the command line to make that work?  Do I type `labpdfproc --u toast=for breakfast` or do I have to type `labpdfproc --u toast="for breakfast"`

On the whole, if we are going to go with the approach of parsing strings and looking for "=" it would be better if we used `nargs='+'` and allowed multiple key-value pairs to be passed at the same time (this feels more intuitive to me as a user), then allow the user to escape the = with a slash, for example.  We then need to give very good instructions in the help, e.g., see here https://stackoverflow.com/questions/27146262/create-variable-key-value-pairs-with-argparse-python